### PR TITLE
Update server implementation per gribi proto devel-v1.5.0

### PR DESCRIPTION
**Pull request by xw-g**
* (M) go.mod
* (M) go.sum
  - Update gribi module per https://github.com/openconfig/gribi/releases/tag/devel-v1.5.0
 
* (M) server/server.go
* (M) server/server_test.go
* (M) compliance/flush.go
  - Update Flush() response per https://github.com/openconfig/gribi/pull/45
 
* (M) rib/rib_test.go
  - Update test cases per https://github.com/openconfig/gribi/pull/48

### CODE REVIEW

**Comment by xw-g** [comment]
cc: @liulk

File: rib/rib_test.go [hunk 1]
```diff
@@ -3358,10 +3358,8 @@ func TestFlush(t *testing.T) {
 			r := NewRIBHolder("DEFAULT")
 
 			_, _, err := r.AddNextHop(&aftpb.Afts_NextHopKey{
-				Index: 1,
-				NextHop: &aftpb.Afts_NextHop{
-					ProgrammedIndex: &wpb.UintValue{Value: 42},
```
Comment (robshakir) [inline rib/rib_test.go:3363]: Here and below, is there another field that we can populate within the `NextHop` just to check that it is being populated correctly and returned in the RIB?

**Review by robshakir — APPROVED** [review]
One change just to ensure that we don't lose coverage of setting fields in particular types.

File: rib/rib_test.go (hunk 1 above)
Comment (xw-g) [reply rib/rib_test.go:3363]: Done.

Comment (xw-g) [reply rib/rib_test.go:3363]: Done

**Pull request merged**